src/modelframe.jl

```diff
@@ -1,4 +1,6 @@
 """
+    ModelFrame(formula, data; mod=StatisticalModel, contrasts=Dict())
```

Comment:
How about renaming `mod` to `model`? It matters since that's a keyword argument.

Reply:
I thought `ModelFrame` was getting deprecated...

Reply:
not until later...trying to keep the changes as minimal as possible here.

Reply:
K, just to know how to develop for the new area. I don't wanna DataTables myself again. Right after the min changes, a deprecation one might be a good step.

Reply:
yeah of course; see what @dmbates has done with MixedModels.jl.  There's some advice in the docs too.  Short version is to use `schema`, `apply_schema`, and `modelcols` directly instead of `ModelFrame`/`ModelMatrix`.